WowPacketParser/Loading/SniffFile.cs

```diff
@@ -248,12 +256,19 @@ private void ProcessFileImpl()
 
                             // Close Writer, Stream - Dispose
                             packet.ClosePacket();
+                            packets.Packets_.Add(packet.Holder);
```

Comment:
should this be guarded by a if check so it gets filled only when needed, i.e. when _dumpFormat == DumpFormatType.UniversalProto ?

Reply:
Right, only in that one case there is point in adding the packet to the list

Reply:
you could set a bool variable "storePackets" or so with the _dumpFormat == DumpFormatType.UniversalProto check so in the future it's easy to change the logic only in that place at the beginning